### PR TITLE
[New Model]: QualityMessage 1.0.0

### LINKED ISSUE #305: [New Model]: QualityMessageContent
## Model Description
Notifications are - in contrast to classical data offers - a way to push data from a sender to a receiver. This model describes the message content for quality investigations and alerts. It contains standardized attributes for message content common across most use cases in CatenaX. It belongs to the same group of aspects as the MessageHeader Aspect Model: io.catenax.shared.message_header 

## Example Data
},
  "content": {
         "notificationId": "c2801472-5f87-41a7-9a25-b0939c4e0dff",
         "status": "SENT",
         "severity": "CRITICAL",
         "information": "Gear boxes loose oil while driving.",
         "listOfAffectedItems": [
             "urn:uuid:57e4e3c1-a6f0-46a0-90df-1fb17cbc157d",
             "urn:uuid:e4da568b-8cf1-4f5f-a96a-cf26265b2c72"
    ]
  }
}

## MS1 Criteria

- [x] The proposed aspect model does not exist already
- [x] The proposed aspect model does not extend an existing aspect but introduces completely new functionality
- [x] The proposal references a Catena-X use case
- [x] Relevant standards are mentioned/linked
- [x] A first draft of the data structure is provided in an example

### PR DESCRIPTION
**Pull request by johannsvarela**
## Description
Notifications are - in contrast to classical data offers - a way to push data from a sender to a receiver. This model describes the message for quality investigations and alerts. It contains standardized attributes for message content common across most use cases in CatenaX. It belongs to the same group of aspects as the MessageHeader Aspect Model: io.catenax.shared.message_header

The bugfix (#476) of the shared Message Header aspect model must be done before this PR can be continued!

Closes #305 and replaces #311

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [ ] All required reviewers have approved this PR (see reviewers section)
- [ ] The new aspect (version) will be implemented by at least one data provider
- [ ] The new aspect (version) will be consumed by at least one data consumer
- [ ] There exists valid test data
- [ ] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [ ] The model has at least version '1.0.0'
- [ ] If a previous model exists, model deprecation has been checked for previous model
- [ ] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by agg3fe** [comment]
@johannsvarela is this going to be common/shared between other models also? If yes, then please change the package structure to io.catenax.shared.--, like for shared message header model.

**Comment by johannsvarela** [comment]
modeling_team

**Comment by johannsvarela** [comment]
FYI @jonbckr @BenediktMuellerIML @jacewski-bosch

**Comment by agg3fe** [comment]
> @johannsvarela is this going to be common/shared between other models also? If yes, then please change the package structure to io.catenax.shared.--, like for shared message header model.

This is not shared.

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by agg3fe** [comment]
@johannsvarela could you please update your branch with main branch. As your code does not include message header 2.0.0, so it is failing.

**Comment by johannsvarela** [comment]
@agg3fe Done! :)

**Comment by agg3fe** [comment]
> @agg3fe Done! :)

Thanks. assigned MS2 again. Let's do MS3 together in next meeting.

File: io.catenax.quality_message/1.0.0/QualityMessage.ttl [hunk 1]
```diff
@@ -0,0 +1,126 @@
+######################################################################
+# Copyright (c) 2023 BASF SE
```
Comment (catroest) [inline io.catenax.quality_message/1.0.0/QualityMessage.ttl:2]: 2023-2024

File: io.catenax.quality_message/RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Changelog
+All notable changes to this model will be documented in this file.
+
+## [Unreleased]
+
+## [1.0.0] - 2023-11-27
```
Comment (catroest) [inline io.catenax.quality_message/RELEASE_NOTES.md:6]: date

**Comment by johannsvarela** [comment]
@agg3fe @catroest No required anymore, therefore this PR will be closed by myself! :)

**Pull request closed without merging**